linode/terraform-provider-linode
# Fix `group` Attribute Deletion Issue in `linode_instance`

## 📝 Description

closes #1296 

Bump linodego and modify code accordingly to fix the bug in `group` attribute deletion.

## ✔️ How to Test

### Automated Testing

`make PKG_NAME=linode/instance int-test`

### Manual Testing

- Create a Linode instance resource with `group` attribute, `terraform apply`.
```terraform
resource "linode_instance" "example" {
  image           = "linode/debian12"
  label           = "testing-group-delete"
  region          = "us-mia"
  type            = "g6-nanode-1"
  group           = "test-remove"
}
```

- Remove `group` attribute and update the resource, `terraform apply`.

```terraform
resource "linode_instance" "example" {
  image           = "linode/debian12"
  label           = "testing-group-delete"
  region          = "us-mia"
  type            = "g6-nanode-1"
}
```

- Verify if the plan is empty, `terraform plan`. It should show nothing to change.
```
No changes. Your infrastructure matches the configuration.
```
